### PR TITLE
Fix SQS and Kafka inbound reference documentations

### PR DESCRIPTION
**Pull request by arunans23**
This PR resolves the following issues.
- https://github.com/wso2/docs-mi/issues/637
- https://github.com/wso2/docs-mi/issues/625

### CODE REVIEW

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 1]
```diff
@@ -33,14 +33,14 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
...
-        <tr>
-            <td>topic.pattern</td>
-            <td>The name pattern of the topic.</td>
-        </tr>
         <tr>
             <td>contentType</td>
             <td>The content type of the message.</td>
         </tr>
+        <tr>
+            <td>topic.partitions</td>
```
Comment (SanojPunchihewa) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md:41]: This is an optional parameter. Shall we move it under **Optional parameters for Kafka Inbound Endpoint**

File: en/docs/reference/connectors/amazonsqs-connector/amazon-inbound-endpoint-1.0.x/amazonsqs-inbound-endpoint-reference-configuration.md
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/amazon-inbound-endpoint-1.0.x/amazonsqs-inbound-endpoint-reference-configuration.md:1]: - Shall we remove the additional styling elements from the table and make it consistent with the rest of the tables used in the doc?
- When mentioning parameter names, and values, please use backticks or `<code></code>` tag if it's inside a HTML table.

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-reference-configuration.md
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-reference-configuration.md:1]: - Shall we remove the additional styling elements from the table and make it consistent with the rest of the tables used in the doc?
- When mentioning parameter names, and values, please use backticks or `<code></code>` tag if it's inside a HTML table.

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md:1]: When mentioning parameter names, possible values, etc., please use backticks or <code></code> if it's inside a HTML table.

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md (hunk 1 above)
Comment (arunans23) [reply en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md]: Fixed.

File: en/docs/reference/connectors/amazonsqs-connector/amazon-inbound-endpoint-1.0.x/amazonsqs-inbound-endpoint-reference-configuration.md
Comment (arunans23) [reply en/docs/reference/connectors/amazonsqs-connector/amazon-inbound-endpoint-1.0.x/amazonsqs-inbound-endpoint-reference-configuration.md:1]: Fixed.

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-reference-configuration.md
Comment (arunans23) [reply en/docs/reference/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-reference-configuration.md:1]: Fixed.

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md
Comment (arunans23) [reply en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md:1]: Fixed

File: en/docs/reference/connectors/amazonsqs-connector/amazon-inbound-endpoint-1.0.x/amazonsqs-inbound-endpoint-reference-configuration.md [hunk 2]
```diff
@@ -10,90 +10,83 @@ The following configurations allow you to configure AmazonSQS Inbound Endpoint f
 </style>
 <table class="tg">
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/amazon-inbound-endpoint-1.0.x/amazonsqs-inbound-endpoint-reference-configuration.md:11]: Let's remove this class name and the style tags.

File: en/docs/reference/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-reference-configuration.md [hunk 3]
```diff
@@ -13,90 +13,83 @@ The following configurations allow you to configure AmazonSQS Inbound Endpoint f
 </style>
 <table class="tg">
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-reference-configuration.md:14]: Let's remove this class name and the style tags.

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 4]
```diff
@@ -10,35 +10,31 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
             <th>Description</th>
         </tr>
         <tr>
-            <td>bootstrap.servers</td>
+            <td><code>bootstrap.servers</code></td>
             <td>The Kafka brokers listed as host1:port1 and host2:port2</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md:14]:
```suggestion
            <td>The Kafka brokers listed as <code>host1:port1</code> and <code>host2:port2</code></td>
```

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 5]
```diff
@@ -10,35 +10,31 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
...
         <tr>
-            <td>bootstrap.servers</td>
-            <td>The Kafka brokers listed as host1:port1 and host2:port2</td>
+            <td><code>bootstrap.servers</code></td>
+            <td>The Kafka brokers listed as <code>host1:port1</code> and <code>host2:port2</code></td>
         </tr>
         <tr>
-            <td>key.deserializer</td>
+            <td><code>key.deserializer</code></td>
             <td>Deserializer class for key that implements the org.apache.kafka.common.serialization.Deserializer interface.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md]:
```suggestion
            <td>Deserializer class for key that implements the <code>org.apache.kafka.common.serialization.Deserializer</code> interface.</td>
```

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 6]
```diff
@@ -10,35 +10,31 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
...
         <tr>
-            <td>key.deserializer</td>
-            <td>Deserializer class for key that implements the org.apache.kafka.common.serialization.Deserializer interface.</td>
+            <td><code>key.deserializer</code></td>
+            <td>Deserializer class for key that implements the <code>org.apache.kafka.common.serialization.Deserializer</code> interface.</td>
         </tr>
         <tr>
-            <td>value.deserializer</td>
+            <td><code>value.deserializer</code></td>
             <td>Deserializer class for value that implements the org.apache.kafka.common.serialization.Deserializer interface.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md:22]:
```suggestion
            <td>Deserializer class for value that implements the <code>org.apache.kafka.common.serialization.Deserializer</code> interface.</td>
```

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 7]
```diff
@@ -52,333 +48,343 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
...
+            <td></td>
+        </tr>
+        <tr>
+            <td><code>topic.partitions</code></td>
+            <td>A comma-separated list of partitions for the topic to which the consumer is subscribed.</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td><code>enable.auto.commit</code></td>
             <td>Whether the consumer will automatically commit offsets periodically at the interval set by auto.commit.interval.ms.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md:62]:
```suggestion
            <td>Whether the consumer will automatically commit offsets periodically at the interval set by <code>auto.commit.interval.ms</code>.</td>
```

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 8]
```diff
@@ -52,333 +48,343 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
...
         <tr>
-            <td>basic.auth.credentials.source</td>
+            <td><code>basic.auth.credentials.source</code></td>
             <td>Specify how to pick the credentials for the Basic authentication header.</td>
             <td></td>
         </tr>
         <tr>
-            <td>basic.auth.user.info</td>
+            <td><code>basic.auth.user.info</code></td>
             <td>Specify the user info for the Basic authentication in the form of {username}:{password}.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md:112]:
```suggestion
            <td>Specify the user info for the Basic authentication in the form of <code>{username}:{password}</code>.</td>
```

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 9]
```diff
@@ -52,333 +48,343 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
...
-            <td>basic.auth.user.info</td>
-            <td>Specify the user info for the Basic authentication in the form of {username}:{password}.</td>
+            <td><code>basic.auth.user.info</code></td>
+            <td>Specify the user info for the Basic authentication in the form of <code>{username}:{password}</code>.</td>
             <td></td>
         </tr>
         <tr>
-            <td>ssl.key.password</td>
+            <td><code>ssl.key.password</code></td>
             <td>The password of the private key in the key store file or the PEM key specified in `ssl.keystore.key`.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md:117]:
```suggestion
            <td>The password of the private key in the key store file or the PEM key specified in <code>ssl.keystore.key</code>.</td>
```

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 10]
```diff
@@ -52,333 +48,343 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
...
         <tr>
-            <td>ssl.keystore.location</td>
+            <td><code>ssl.keystore.location</code></td>
             <td>The location of the key store file. This is optional for client and can be used for two-way authentication for client.</td>
             <td></td>
         </tr>
         <tr>
-            <td>ssl.keystore.password</td>
+            <td><code>ssl.keystore.password</code></td>
             <td>The store password for the key store file. This is optional for client and only needed if ‘ssl.keystore.location’ is configured.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md:127]:
```suggestion
            <td>The store password for the key store file. This is optional for client and only needed if <code>ssl.keystore.location</code> is configured.</td>
```

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 11]
```diff
@@ -52,333 +48,343 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
...
         <tr>
-            <td>sasl.jaas.config</td>
+            <td><code>sasl.jaas.config</code></td>
             <td>JAAS login context parameters for SASL connections in the format used by JAAS configuration files.</td>
             <td></td>
         </tr>
         <tr>
-            <td>sasl.client.callback.handler.class</td>
+            <td><code>sasl.client.callback.handler.class</code></td>
             <td>The fully qualified name of a SASL client callback handler class that implements the AuthenticateCallbackHandler interface.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md]:
```suggestion
            <td>The fully qualified name of a SASL client callback handler class that implements the <code>AuthenticateCallbackHandler</code> interface.</td>
```

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 12]
```diff
@@ -52,333 +48,343 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
...
-            <td>fetch.max.bytes</td>
+            <td><code>fetch.max.bytes</code></td>
             <td>The maximum amount of data the server should return for a fetch request.</td>
-            <td>52428800</td>
+            <td><code>52428800</code></td>
         </tr>
         <tr>
-            <td>max.poll.interval.ms</td>
+            <td><code>max.poll.interval.ms</code></td>
             <td>The maximum delay between invocations of poll() when using consumer group management.</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md]:
```suggestion
            <td>The maximum delay between invocations of <code>poll()</code> when using consumer group management.</td>
```

File: en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md [hunk 13]
```diff
@@ -52,333 +48,343 @@ The following parameters are required when configuring Kafka Inbound Endpoint.
...
-            <td>The maximum delay between invocations of poll() when using consumer group management.</td>
-            <td>300000</td>
+            <td><code>max.poll.interval.ms</code></td>
+            <td>The maximum delay between invocations of <code>poll()</code> when using consumer group management.</td>
+            <td><code>300000</code></td>
         </tr>
         <tr>
-            <td>max.poll.records</td>
+            <td><code>max.poll.records</code></td>
             <td>The maximum number of records returned in a single call to poll().</td>
```
Comment (DinithiDiaz) [inline en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-config.md:167]:
```suggestion
            <td>The maximum number of records returned in a single call to <code>poll()</code>.</td>
```

**Comment by DinithiDiaz** [comment]
LGTM

**Review by chanikag — APPROVED** [review]

**Pull request merged**